src/client/dfs/README.md

```diff
@@ -242,3 +242,60 @@ pool):
  - SX if rd\_fac == 0
  - EC\_nP1GX if rd\_fac == 1
  - EC\_nP2GX if rd\_fac == 2
+
+# DFS control flow
+
+Below are a few sequence diagrams to illustrate what happens when a dfs API is
+called.
+
+## Opening an existing file with dfs_open
+```mermaid
+sequenceDiagram
+    box Client
+        participant A as application
+        participant B as libdfs
+        participant C as libdaos
+    end
+    box Server
+        participant D as parent RG member
+    end
+
+    A->>B: dfs_open
+    B->>C: daos_obj_fetch(parent_obj)
```

Comment:
does it make more sense to indicate at this level that the fetch is conditional (since the flag is set by libdfs)?
can we modify this or add a comment of what we are fetching?
at first sight this might indicate for a user that we are fetching the parent object.. so it's not clear that what we are fetching here is the file (inode) entry from the parent object..

Reply:
thanks. might also want to push there with doc-only pragma

Reply:
ah nevermind, our CI recognizes that